config/initializers/devise.rb

```diff
@@ -151,66 +101,6 @@
   # time the user will be asked for credentials again. Default is 30 minutes.
   config.timeout_in = 6.hours
 
-  # ==> Configuration for :lockable
```

Comment:
I explored implementing lockable in #1697. Turns out lockable doesn't work as a drop-in replacement for us because we're using our own ldap_authenticable_tiny strategy instead of database_authenticable. So my plan is to not use Devise's out-of-the box lockable at all and roll our own implementation.